### DISCUSSION TITLE
The deleted sectors on the chain still display Faulty Sectors

### DISCUSSION THREAD

**Discussion by lhjok** [filecoin-project/lotus#9836] [General Q&A]
**Hello everyone**

`lotus-miner info (Sector overview)
`
```
Sectors:
	Total: 5970
	Proving: 5710
	PreCommit1: 14
	PreCommit2: 1
	SubmitPreCommitBatch: 3
	Committing: 10
	SubmitCommitAggregate: 9
	Removed: 223


Workers: Seal(7) WdPoSt(0) WinPoSt(0)

Storage Deals: 0, 0 B

Retrieval Deals (complete): 0, 0 B
```

**- The above information shows that the sectors on the chain have been successfully deleted**

```
Deadline Index:           3
Partitions:               1
Proven Partitions:        0
Current:                  false

Partition Index:          0
	All Sectors:               969
	Live Sectors:              968
	Active Sectors:            927
	Faulty Sectors:            1
	Recovering Sectors:        0
```

Various methods have been tried, including creating new sector files. 
The sector **_cannot_** be deleted.

Any help is appreciated.

**Comment by Reiers** [comment]
**Hi @lhjok** 

**Have you terminated them onChain?**
If it on chain, after Removing you should Terminate them:

`lotus-miner sectors terminate --really-do-it <sectornumber>
`

If that doesn't work you can use `lotus-shed`
```
cd lotus/
make lotus-shed
./lotus-shed sectors terminate --really-do-it=true sectorNumbers
```
ONLY if the sector is unrecoverable.
